src/OneExplorer/OneExplorer.ts

```diff
@@ -500,7 +499,17 @@ input_path=${modelName}.${extName}
 
 export function initOneExplorer(context: vscode.ExtensionContext) {
   // TODO Support multi-root workspace
-  let workspaceRoot: vscode.Uri = vscode.Uri.file(obtainWorkspaceRoot());
+  let workspaceRoot: vscode.Uri|undefined = undefined;
+  try {
+    workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      Logger.error('OneExplorer', e.message);
+    } else {
+      throw e;
+    }
+  }
```

Comment:
I changed the way to handle the error. How about this way? /cc @dayo09

Reply:
As @jyoungyun says above, I think checking whether it's an instance of Error doens't carry any information here.. I think you can decide here is the treatment for whatever cases. - "whether to throw or log"..

Reply:
```suggestion
  try {
    workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
  } catch (e: unknown) {
    if (e instanceof Error && e.message == 'Need workspace') {
      Logger.error('OneExplorer', e.message);
    } else {
      throw e;
    }
  }
```

@dayo09 @jyoungyun This change can check whether the error is what we expected or not. If the error is expected no-workspace error, it's okay to keep going. 
IMHO break because of unexpected error is a feature need to have. How do you think of this approach?

Reply:
I agree with the intention. I think it will be clearer if an Error is specifically defined from `obtainWorkspaceRoot`.

----
+ About throwing an error

I just checked what happens when OneExplorer throws: The whole extension throws. 

![0714-throw](https://user-images.githubusercontent.com/17171963/178684205-a7033ab5-287a-4f86-adb3-25d534cce5e2.png)

So an expected error from OneExplorer shuts down the whole extension, including ToolchainView and TargetDeviceView....
Maybe it's a right thing to do...? I don't know.

Reply:
```ts
  try {
    workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
  } catch (e: unknown) {
    if (e instanceof Error) {
      if (e.message == 'Need workspace') {
        Logger.info('OneExplorer', e.message);
      } else {
        Logger.error('OneExplorer', e.message);
        Balloon.error('Something goes wrong while setting workspace.', true);
        // throw e;
      }
    } else {
      Logger.error('OneExplorer', 'Unknown error has been thrown.');
      // throw e;
    }
  }
```

This version handles unexpected error, and print the separated logs for each types. I don't know either wheter throwing is right thing to do..

Reply:
Agree. Anyways think we can talk about it later on.